tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SNS/AwsSnsCommon.cs

```diff
@@ -0,0 +1,60 @@
+// <copyright file="AwsSnsCommon.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
```

Comment:
nit: we try to create new files with `#nullable enable` set - ideally this would be added to all the new files

```suggestion

#nullable enable
```